### DISCUSSION TITLE
Outside communities rating eachother's games highly / games not being exposed to random audience

### DISCUSSION THREAD

**Discussion by BurgeeGames** [ludumdare/ludumdare#2064] [General]
There's an issue on the front page of LDJam.com right now and an ongoing discussion in the comments of the game in question:

https://ldjam.com/events/ludum-dare/41/war-maker/this-is-my-first-and-last-ld
https://ldjam.com/events/ludum-dare/41/are-you-smarter-than-an-alien (Not trying to call him out)

The issue is the author is a member of an outside game dev community whose friends commented in his game stating that they were 'outside friends' and stating that they rated '5/5'. To compound the point, the same person admits others from his group rated the game, and to even further compound the issue, the game has exactly 20 ratings which people tend to find suspicious.

I'm not accusing anyone of cheating here, but I am concerned that games may or may not be exposed to outside audiences if the authors 1.) Don't rate other games and 2.) Link it to their friends and link it externally on dev sites.

Again, ratings at the end of the day don't really mean much and it's the feedback that counts, but people are upset over this, their LD experience is ruined and a dev that took high marks without trying to game the system may be feeling bad about his wins. 

Having a filter for games with 15-25 ratings might be a good fix, since people tend to find those numbers suspicious and would seek them out / police the issue themselves, and since many many winners in the past couple of years have had rating numbers close to the minimum amount needed.

**Comment by mikekasprzak** [comment]
This game's suspicios ratings have been brought up to me outside of this,
and it's something I was hoping wouldn't turn in to a bigger problem. Alas.

It does bring up an interesting point we could push in the future: try to
avoid leaving ratings on your friends games (or internally flag them as
something to ignore for placement). I'm not entirely sure how we do or
enforce that, but a large enough group of friends (classmates) are able to
damage the feeling of fairness.

**Comment by ajayyy** [comment]
The only issue is people might define friends as different things. Is rating other people you talk to frequently in the Ludum Dare discord, for example, considered rating a friends game?

But I guess leaving it up to the person's moral judgement (like which is done for many rules) to decide might be the best way, but just suggestion not to do it if you feel it won't be fair.

**Comment by samryl** [comment]
During LD38 I heard someone put forward an excellent suggestion for something of a ‘homework’ list:

    Each user gets a list of ~10 games. Each list is unique to user, so no one game is in two users’ queues at the same time.
        When the user rates the game, it is removed from their list and added to another users’ list.
        When the game gets enough votes to rank, it is removed entirely from all the queues.
    If a game sits in a queue long enough (2-3 days?) it gets moved to another queue.

Of course you can still browse games and play any that you want but this helps distribute all the games between all the users.

As for rating itself, instead of rating and then commenting, perhaps the page should make you put in a short comment for each category. This would at least encourage more constructive, detailed feedback and we could see the definable extent of the blanket/buddy voting problem before taking further action against it.

**Comment by ajayyy** [comment]
@Samusoidal #954

**Comment by samryl** [comment]
Thanks @ajayyy, had no idea that existed!

**Comment by jcmonkey** [comment]
i left this for viewing in a post on the LD site, but ill repost my opinion here too. 

here is my simple example of how one can cheat and game the system, and i hope this gets fixed in the future

1) make 20+ ludum dare accounts and have each of them join the jam ( thus starting the exploitation )

2) dont play any games, until the next week end, this ensures no game visibility and no ratings

3) rate your game 4.5 or higher in all categorys with your 20+ accounts you premade before jam

4) you can then now rate and play other games to make your account look normal

after the first week most ppl stop rating and playing, so you are good
5) profit, because your game will score top 10, then you can email websites and get mad press about your game

i hope i was able to get my point across appropriately here enough that something can be done about this.

thanks

**Comment by samryl** [comment]
Some quick solutions:
 - Unique email verification -> Deters people from creating multiple accounts
 - Can't vote unless you published -> Deters people with multiple accounts from giving unfair ratings
 - Can't publish without a link -> Deters people from publishing fake games in order to give unfair ratings

There is of course a loophole in all of these. No solution to such a huge, community-driven event will be airtight. There will always be someone who might be able to put in the time and effort to circumvent the rules.

**Comment by occultsoftworks** [comment]
These are good suggestions, I'm gonna poke holes in them only for the purposes of strengthening our possible solutions if you don't mind.

Unique Emails: This is not a deterrent to a bad actor. It's ~1 minute or less to acquire a new gmail. A stronger form of this might be tying a Facebook account to your account, but not everyone uses facebook and some folks like anonymity or live in countries where being de-anonymized could get them in trouble. The worst from a design standpoint but strongest form of security of this might be tying a credit card or some third party verified number to the system. This would kill the amount of newcomers coming in, but it would certainly be a major weedout.

Can't Vote Unless Published: I actually agree with this one. I will say, nothing is stopping a determined actor from making junk games to put up ahead of time so it's not foolproof. I will caveat this with: It would suck for the youtube community that has sprung up around LD.

Can't publish without a link: I feel like this might really hamper board games. And seriously, nothing is stopping someone from throwing up a "Broken" game. Conversely, we don't want to hamper people who tried and failed to make a game, failed to submit, but want to be part of the community.

**Comment by samryl** [comment]
Definitely good points, @occultsoftworks, and ones I though of myself. 
I think 'Unique Emails' is the strongest point, flawed as it is. You'd have to really want to cheat to create a new mailbox and LD account. This would help deter 'passive cheaters.' Plus, if email ever comes into play in later functionality of the site, we'll have a system in place.
As for the YouTube voting community, should they be voting? LD is all about 'By the Community, For the Community,' but is it fair to have non-participants voting? I'm not sure at all.
I'd like to keep the angle of actual systems in place to prevent cheating rather than the 'fairness rules change' angle, but I though I should mention it.

**Comment by occultsoftworks** [comment]
"Should YouTuber's be given free status to vote" is a "The Heart and Soul of Ludum Dare" question. Probably outside the scope of this specific conversation. Might come down to how Mike K. feels about it.

**Comment by occultsoftworks** [comment]
I had a thought: Unique Emails only creates a "Better Class of ~Criminal~ Cheater". It adds friction between users who may have a legit reason for a second account (I have one for when I do compo entries personally without the rest of my team) and their desired outcome of a second account, and if someone /is/ cheating, they might slip up and could be tracked thru those duplicate emails.

**Comment by samryl** [comment]
I don't see the reason for a second account. Currently, each member of a team can create an account and be added to Jam entry as 'team members' by a single person. Alternatively, that person could choose not to form a team and instead publish a Compo game.

If the user is publishing two games (Jam [Team] and Compo?), this problem is resolved by having a different team member publish the game and add users to its team.

Are there any other legitimate reasons for a second account that I'm missing?

**Comment by mikekasprzak** [comment]
For reference:
* Email accounts _are_ tied to a single account (you can't use the same account twice, though the workaround is if you know about GMail's `+` feature)
  * In the future we will allow multiple e-mail accounts on the same account, but this is primarily for migration.
* Throwaway email providers were blacklisted a few days before LD41 event due to someone abusing them to leave hate messages
* You must be published to vote (this has always been a requirement)
  * The workaround for a while was that a published `unfinished` game was counted as published. I seem to remember putting a constraint in that stopped this at some point, but I haven't checked recently.
* Link checking is a good idea
  * In general we need better link features. A verifier, way to report bad links (to the author), tracking (did you even click on a download before voting), etc.
* We do need to add a constraint that stops you from selecting the compo if you have multiple members
* We would also need to add something that raises a flag/popup if you try to add users to your compo game (to move it to the jam)
* Team features need to be improved. The limit will still be 1 game per person, but if you can be a contributor to any number of games. Problem is it's not obvious how to edit or blog under each game.
* IP checking will not work since people at the same jam site will share the same IP (though it could be used it to raise flags internally if we start tracking suspicious behavior)

**Comment by local-minimum** [comment]
After reading the posts on ldjam.com my two cents are:

* I really don't think voter-rings in itself is a very common or problematic abuse compared to other forms of abuse such as quick rating to promote own game -- with or without posting comments like 'cool game play mine [link]'. I think the more serious disruptive aspect here is the backlash from the perceived abuse.
* However, everyone probably benefits from getting feedback from non-friends and it would be good if there could be something done to limit the risk of winners being perceived as a result from friends-promoted games.

I have a suggestion for solving this that should be rather simple and ties back to various other issues discussed for other things: An update to the `smart` filter.

__I suggest we inject a low percentage of very highly rated games into the smart-feed.__

One very simple rule would be the sum vote averages for the three best categories of each game. This could be weighted by number of votes so that it has weight 1 if number of votes is 20 and decreasing towards 0 when having more or less than 20 votes.

If say 5% of the first 20 games you see are from this criteria rather than the 'most deserving'-criteria and you don't know which is which as a rater (unless you study the game's stats), I think this could have several beneficial effects:

* Voters may be inclined to vote for more games because they will stumble upon really interesting games more often and since the best-games criteria is weighted around 20 votes, it still wouldn't be just a few games get all the votes.
* If someone is indeed trying to abuse the system, their game will automatically get an outside-vote or two thanks to high ratings. And a single angry vote will have a large effect on the average when number of votes is low. So it would more or less guarantee games are really deserving of winning/ranking top 20ish.
* It would also mean we start including functionality into the smart filter (mixing in of criteria) that we've discussed previously as something we should have.
